app/services/models_services/creators/bookings.rb

```diff
@@ -0,0 +1,81 @@
+module ModelsServices
+  module Creators
+    class Bookings
```

Comment:
A namespace called `ModelsServices` inside the servces folder feels very weird. What makes a service class be inside or outside ModelsService?

I think you are not using namespaces here in the correct manner. Services are usually meant to be there to store business logic. I think it's better if you think of namespaces inside services as a way to separate business domains that can stand on their own, so it's better if you group classes in namespaces according to their domain, not what they do. Hence, I think it's better if you have a `Bookings` namespace that will contain everything related to the bookings business domain, such as creation, confirmation, modification, cancelation, etc. So as your application grows, you can just keep adding more namespaces as business domains grow, for example, a separate `Checkout` or `Authentication` service. Defining this correctly in real apps is part of defining the backend architecture, and by doing it this way you can have namespaces grouping classes that are more related to each other and more independent from other namespaces.

So this class could be `Bookings::Creator` or `Bookings::BookingCreator`

Reply:
Done!